plugin/build.gradle.kts

```diff
@@ -37,7 +37,7 @@ versionCompatibility {
     tests {
         dimensions {
             register("Gradle") {
-                versions = listOf("7.0", "7.1", "7.2", "7.3", "7.4", "7.5.1", "7.6", "8.0.2", "8.1.1", "8.2.1", "8.3", "8.4-rc-3")
+                versions = listOf("7.0", "7.1", "7.2", "7.3", "7.4", "7.5.1", "7.6", "8.0.2", "8.1.1", "8.2.1", "8.3", GradleVersion.current().version)
```

Comment:
This must be hard-coded to ensure that all Gradle versions that have ever been tested will continue to be tested. Otherwise, the compatibility test for 8.4 will be dropped when 8.5 comes around.

Reply:
02ae5c12f8deb0ec041a3166261f088c52354b19